lib/core/base/context/parse-selector-array.js

```diff
@@ -12,29 +12,20 @@ export function parseSelectorArray(context, type) {
   const result = [];
   for (let i = 0, l = context[type].length; i < l; i++) {
     const item = context[type][i];
-    // selector
-    if (typeof item === 'string') {
-      const nodeList = Array.from(document.querySelectorAll(item));
-      result.push(...nodeList.map(node => getNodeFromTree(node)));
-      break;
-    }
-
     // Handle nodes
     if (item instanceof window.Node) {
       if (item.documentElement instanceof window.Node) {
         result.push(context.flatTree[0]);
       } else {
         result.push(getNodeFromTree(item));
       }
-      continue;
-    }
 
-    // Handle Iframe selection
-    if (item && item.length) {
+      // Handle Iframe selection
+    } else if (item && item.length) {
```

Comment:
This might just as well be "else". If this isn't an array, normalization has a bug. Open to suggestions.